include/sys/__assert.h

```diff
@@ -4,60 +4,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
```

Comment:
Ah, I was wondering if this was going to be duplicated.  Removing it here and adding to the docs eliminates that potential inconsistency problem.